contrib/docker/docker-compose.prod.yaml

```diff
@@ -38,6 +38,7 @@ services:
       JITSI_URL: $JITSI_URL
       JITSI_PRIVATE_MODE: "$JITSI_PRIVATE_MODE"
       PUSHER_URL: //pusher.${DOMAIN}
+      PUSHER_URL: //icon.${DOMAIN}
```

Comment:
typo? did you mean `ICON_URL`?

Reply:
Ping @nolway

Reply:
Ping @nolway The problem is still here it seems :)

Reply:
```suggestion
      ICON_URL: //icon.${DOMAIN}
```